pallets/connectors/src/message.rs

```diff
@@ -201,6 +201,39 @@ where
 		tranche_id: TrancheId,
 		investor: Address,
 	},
+	/// The message sent back to the domain from which a `CollectInvest` message
+	/// has been received, which will ensure the `investor` gets the payout
+	/// respective to their investment.
+	///
+	/// Directionality: Centrifuge -> EVM Domain.
+	ExecutedCollectInvest {
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		investor: Address,
+		/// The currency in which the payout takes place
+		currency: u128,
+		/// The amount being paid out to the investor
+		payout: Balance,
+		/// The remaining amount the investor still has locked
+		remaining_locked_investment: Balance,
+	},
+
+	/// The message sent back to the domain from which a `CollectRedeem` message
+	/// has been received, which will ensure the `investor` gets the payout
+	/// respective to their redemption.
+	///
+	/// Directionality: Centrifuge -> EVM Domain.
+	ExecutedCollectRedeem {
+		pool_id: PoolId,
+		tranche_id: TrancheId,
+		investor: Address,
+		/// The currency in which the payout takes place
+		currency: u128,
+		/// The amount being paid out to the investor
+		payout: Balance,
+		/// The remaining amount the investor still has locked
+		remaining_locked_redeem: Balance,
```

Comment:
I agree with @offerijns that we should keep the naming consistent, e.g. one of the following

> * `remaining_locked_investment` / `remaining_locked_redemption`
> * `remaining_locked_invest` / `remaining_locked_redeem`